app/src/main/java/io/homeassistant/companion/android/util/vehicle/DomainCheck.kt

```diff
@@ -0,0 +1,19 @@
+package io.homeassistant.companion.android.util.vehicle
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.common.data.integration.domain
+import io.homeassistant.companion.android.vehicle.MainVehicleScreen
+
+@RequiresApi(Build.VERSION_CODES.O)
+fun isVehicleDomain(entity: Entity<*>): Boolean {
+    return entity.domain in MainVehicleScreen.SUPPORTED_DOMAINS ||
+        entity.domain in MainVehicleScreen.NOT_ACTIONABLE_DOMAINS ||
+        (entity.domain in MainVehicleScreen.MAP_DOMAINS && canNavigate(entity))
+}
+
+fun canNavigate(entity: Entity<*>): Boolean {

```

Comment:
Missing check: `domain == "sensor"`?

Reply:
This check applies to any domain that needs to be navigable

Reply:
But is any domain other than sensor expected? Otherwise this also needs a docs update to reflect that it can be any entity with these attributes.

Reply:
its not any domain with a sensor, only the map domains. Let me make the call more clear :)

Reply:
This function isn't checking for map domains though, only attributes

Reply:
this check now considers the domain as well :)